projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -53,15 +53,15 @@ public static function init() {
 		add_action( 'deleted_plugin', array( __CLASS__, 'deleted_plugin' ), 10, 2 );
 
 		add_action( 'rest_api_init', array( __CLASS__, 'add_is_managed_extension_field' ) );
-		add_action( 'rest_api_init', array( Scheduled_Updates_Logs::class, 'add_log_fields' ) );
-		add_action( 'rest_api_init', array( Scheduled_Updates_Active::class, 'add_active_field' ) );
+		add_action( 'rest_api_init', array( Scheduled_Updates_Logs::class, 'add_log_fields' ), 9 );
+		add_action( 'rest_api_init', array( Scheduled_Updates_Active::class, 'add_active_field' ), 9 );
+		add_action( 'rest_api_init', array( Scheduled_Updates_Health_Paths::class, 'add_health_check_paths_field' ), 9 );
```

Comment:
I wonder if we should move the `rest_api_init` callback in `WPCOM_REST_API_V2_Endpoint_Update_Schedules` to 11 instead, so not every rest field has to be registered at 9 for its schema to make it into the endpoint args.

Reply:
@zaerl Do you have an opinion on this?

Reply:
Yes, it's better. Fewer characters to type 🙂. I don't see any drawbacks.

Reply:
https://github.com/Automattic/jetpack/pull/37223/commits/2b2671a142544b26ed2fc94778db981510408788